include/stumpless/target.h

```diff
@@ -45,16 +45,57 @@
 extern "C" {
 #  endif
 
+/**< write to a character buffer */
+# define STUMPLESS_BUFFER_TARGET_VALUE 0
+
+/**< write to a file */
+# define STUMPLESS_FILE_TARGET_VALUE 1
+
+/**< call a custom function */
+# define STUMPLESS_FUNCTION_TARGET_VALUE 2
+
+/**< send to the systemd journald service */
+# define STUMPLESS_JOURNALD_TARGET_VALUE 3
+
+/**< send to a network endpoint */
+# define STUMPLESS_NETWORK_TARGET_VALUE 4
+
+/**< write to a Unix socket */
+# define STUMPLESS_SOCKET_TARGET_VALUE 5
+
+/**< write to a FILE stream */
+# define STUMPLESS_STREAM_TARGET_VALUE 6
+
+/**< add to the Windows Event Log */
+# define STUMPLESS_WINDOWS_EVENT_LOG_TARGET_VALUE 7
+
+/**
+ * A macro function that runs the provided action once for each target_type,
+ * providing the symbol and value. The action must take two arguments, the
+ * first being the symbol name of the target_type, and the second the numeric
+ * value of the target_type.
+ */
+#  define STUMPLESS_FOREACH_TARGET_TYPE( ACTION )\
```

Comment:
Add the name of this macro function to the header check manifest file as well.

Reply:
by header check manifest you mean `tools/check_headers/stumpless.yml`, `src/windows/stumpless.def` these files ?

Reply:
Only to the `tools/check_headers/stumpless.yml` file. The severity version isn't their either as it was missed in that change, feel free to add it as well.